<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintGraphic</class>
 <widget class="QDialog" name="PaintGraphic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>828</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc.qrc">
    <normaloff>:/icones/Images/Icones/iconeGraph.png</normaloff>:/icones/Images/Icones/iconeGraph.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutScreen">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutComandos">
     <item>
      <widget class="QGroupBox" name="groupBoxParametrosGrafico">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Parâmetros gráfico</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxOrder">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Crescente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Decrescente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aleatório</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonFilterAlgo">
          <property name="font">
           <font>
            <family>Arial</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Filtrar algoritmos</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonFiltrarSize">
          <property name="font">
           <font>
            <family>Arial</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Filtar quantidades</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelOrder">
          <property name="text">
           <string>Ordem dos vetores:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxParametrosTabela">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Parâmetro tabela</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxAlgoritmoTabela"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelAlgoritmoTabela">
          <property name="text">
           <string>Algoritmo na tabela:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxWeb">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Web</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonGeraViewWeb">
          <property name="font">
           <font>
            <family>Arial</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Trabalho teórico</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutPaintGraphic"/>
   </item>
   <item>
    <widget class="QLabel" name="labelCaptionTabela">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Valores do algoritmo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetValues"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxLegenda">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Legenda</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelBubble">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Bubble</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelInsertion">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Insertion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSelection">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMerge">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(39, 234, 255);</string>
        </property>
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelQuick">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Quick</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHeap">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 255);</string>
        </property>
        <property name="text">
         <string>Heap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxOrder</tabstop>
  <tabstop>pushButtonFilterAlgo</tabstop>
  <tabstop>pushButtonFiltrarSize</tabstop>
  <tabstop>comboBoxAlgoritmoTabela</tabstop>
  <tabstop>pushButtonGeraViewWeb</tabstop>
  <tabstop>tableWidgetValues</tabstop>
 </tabstops>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
